src/actions/posts.js

```diff
@@ -679,7 +707,7 @@ export function getPostsAfter(channelId, postId, page = 0, perPage = Posts.POST_
 
         dispatch(batchActions([
             receivedPosts(posts),
-            receivedPostsAfter(posts, channelId, postId),
+            receivedPostsAfter(posts, channelId, postId, posts.next_post_id === ''),
```

Comment:
Using `next_post_id` from API to turn on recent flag on chunk which is used to determine if we have a loader for loading newer posts